lib/host-sp-messages/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "host-sp-messages"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+fletcher = "0.3"
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+
+hubpack = { git = "https://github.com/cbiffle/hubpack", rev = "df08cc3a6e1f97381cd0472ae348e310f0119e25" }
```

Comment:
Do we need to lock this to a commit here, versus in the `Cargo.lock` file?

Reply:
Need to test; I thought that if we had a bare git dependency like

```toml
hubpack = { git = "https://github.com/cbiffle/hubpack" }
```

then cargo would check for updates more or less every time it's run, but maybe that's wrong?